src/App.tsx

```diff
@@ -1,35 +1,56 @@
-
-import CountriesFilter from "./components/controls/CountriesFilter/CountriesFilter";
+import { BrowserRouter as Router, Route, Link } from "react-router-dom";
+import { Routes } from "react-router";
+import RegionDropdown from "./components/controls/RegionDropdown/RegionDropdown";
 import InputSearch from "./components/controls/InputSearch/InputSearch";
 import CountryCard from "./components/sections/CountryCard/CountryCard";
+import CountryDetails from "./components/sections/CountryDetails/CountryDetails";
 import Nav from "./components/sections/Nav/Nav";
 import "./style/style.css";
 
 const App = () => {
 
-  return ( 
-    <div className="App">
-      <Nav/>
-      <main className="main-panel">
-        <section className="main-controls">
-          <InputSearch/>
-          <CountriesFilter/>
-        </section>
-        <section className="countries-panel">
-          <ul className="countries-list">
-            <CountryCard/>
-            <CountryCard/>
-            <CountryCard/>
-            <CountryCard/>
-            <CountryCard/>
-            <CountryCard/>
-            <CountryCard/>
-            <CountryCard/>
-          </ul>
-        </section>
-      </main>
-    </div>
+  return (
+    <Router>
+      <div className="App">
+        <Nav />
+        <main className="main-panel">
```

Comment:
as we discussed - Try to move any HTML tags out of Routing and put Routing in a separate file